execution_engine/src/storage/global_state/in_memory.rs

```diff
@@ -275,18 +271,16 @@ impl StateProvider for InMemoryGlobalState {
         &self,
         _correlation_id: CorrelationId,
         trie_key: &Digest,
-    ) -> Result<Option<Trie<Key, StoredValue>>, Self::Error> {
+    ) -> Result<Option<Bytes>, Self::Error> {
         let txn = self.environment.create_read_txn()?;
-        let ret: Option<Trie<Key, StoredValue>> = self.trie_store.get(&txn, trie_key)?;
+        let ret: Option<Bytes> =
+            Store::<Digest, Trie<Digest, StoredValue>>::get_raw(&*self.trie_store, &txn, trie_key)?
```

Comment:
Why not `self.trie_store.get_raw`?

Reply:
That would work too.

Reply:
Don in https://github.com/casper-network/casper-node/pull/2697/commits/a23f6e6f3b4d9d243f4435b2728ef1830a25fdab

Reply:
Actually, looks like it can't infer the key type when we use just `self.trie_store.get_raw`, so I've had to revert this in 3b4e1c1